README.md

```diff
@@ -194,6 +179,52 @@ From [the new webpack docs](https://webpack.js.org/guides/migrating/#mixing-es20
 
 Unless you pass it the `--plain` param, react-cosmos-webpack runs with `babel-node` by default. This is nice because it allows you to write your configs using the same syntax as your source code.
 
+### Proxies
+
+Proxies are a plugin system for React Cosmos, allowing fixtures to go beyond mocking *props* and *state*. As regular React components, they compose in the order they are listed in your config and decorate the functionality of the loaded component, while respecting the contract to render the next proxy in the chain.
+
+The added functionality can range from mocking Redux state (or server requests made from your components) to creating a resizable viewport for seeing how components behave at different scales.
+
+#### react-cosmos-redux-proxy
+
+Most components in a Redux app depend on Redux state–either they're a *container* or one of their descendants is. This proxy creates the store context required for any component you load, just like [Provider](http://redux.js.org/docs/basics/UsageWithReact.html#passing-the-store) does for your root component. Writing Redux fixtures almost feels too easy. Because Redux state is global, once you have one state mock you can render any component you want!
+
+```js
+// redux-proxy.js
+import createReduxProxy from 'react-cosmos-redux-proxy';
+
+export function () {
```

Comment:
Ouch, by mistake. The examples are [fine](https://github.com/react-cosmos/react-cosmos/blob/d12a20eacdfe44b37f0478ab7e978fdacf501809/examples/context/context-proxy.js#L4).

What I was saying is that you can do
```js
export const createProxy = () => {
```
but not
```js
export default const createProxy = () => {
```

What am I missing?